model/src/test/kotlin/utils/LicenseResolverTest.kt

```diff
@@ -178,8 +178,8 @@ class LicenseResolverTest : WordSpec() {
     }
 
     init {
-        "Given a package with a scan result for VCS, getDetectedLicensesForId()" should {
-            "return all detected licenses" {
+        "getDetectedLicensesForId()" should {
+            "return all detected licenses for a package with a scan result for VCS, " {
```

Comment:
How about "...with a VCS scan result" or "...a scan result from VCS" to avoid the duplicate "for"? Also, the trailing comma should be omitted.

Reply:
Done.